### DISCUSSION TITLE
Is it possible to open more than one window?

### DISCUSSION THREAD

**Discussion by nomad-software** [hajimehoshi/ebiten#2493] [Q&A]
1. Is it possible to open more than one window?
2. If not, is it possible to decide which monitor to open the window on in a multi-monitor setup?

Thanks.

**Comment by hajimehoshi** [comment]
No it is not. You have to create multiple processes.

**Reply by hajimehoshi** [reply]
> If not, is it possible to decide which monitor to open the window on in a multi-monitor setup?

Not yet. There is a task to enable to specify a monitor, but there is not a concrete plan

**Comment by XJIeI5** [comment]
> No it is not. You have to create multiple processes.
I tried to use `ebiten.RunGame()` in goroutine, but the program either did not start, or the error `panic: goglfw: DestroyWindow failed: Access is denied` flew out when closing. Please tell me how I should do it
